package.json

```diff
@@ -74,6 +74,7 @@
     "not op_mini all"
   ],
   "dependencies": {
+    "@cvbb/bc-ur": "^0.2.9",
```

Comment:
Ideally - we don't have this package added to caravan directly, but rather used via `cobovault` in `unchained-wallets`.